jobQueue/setup.js

```diff
@@ -31,6 +31,7 @@ module.exports = {
     BULKUPSERTSAMPLES: 'bulkUpsertSamples',
     PERSIST_SAMPLE_STORE: 'PERSIST_SAMPLE_STORE',
     SAMPLE_TIMEOUT: 'SAMPLE_TIMEOUT',
+    GET_HIERARCHY: 'GET_HIERARCHY',
```

Comment:
pls. insert this in alphabetical order... when there are more of them it will make it easier to read